site/src/pages/ManagementSettingsPage/OrganizationSettingsPage.tsx

```diff
@@ -32,37 +36,49 @@ const OrganizationSettingsPage: FC = () => {
     organizations && organizationName
       ? getOrganizationByName(organizations, organizationName)
       : undefined;
-  const permissionsQuery = useQuery(organizationPermissions(organization?.id));
+  const permissionsQuery = useQuery(organizationsPermissions(organizations));
 
-  if (!organizations) {
+  const permissions = permissionsQuery.data;
+  if (!organizations || !permissions) {
     return <Loader />;
   }
 
-  // Redirect /organizations => /organizations/default-org
+  // Redirect /organizations => /organizations/default-org, or if they cannot edit
+  // the default org, then the first org they can edit, if any.
   if (!organizationName) {
-    const defaultOrg = getOrganizationByDefault(organizations);
-    if (defaultOrg) {
-      return <Navigate to={`/organizations/${defaultOrg.name}`} replace />;
+    const editableOrg = organizations
+      .sort((a, b) => {
+        // Prefer default org (it may not be first).
+        if (a.is_default) {
+          return -1;
+        } else if (b.is_default) {
+          return 1;
+        }
+        return 0;
+      })
```

Comment:
is this duplicated from somewhere? I remember discussing this, but it must've been on a different PR. anyway, the `Number(b.is_default) - Number(a.is_default)` should work here again.